### PR TITLE
Attempt to #572 drag and drop test

### PR DESCRIPTION
**Pull request by Kevin-Wei**
Attempt to seed and see data with poltergiest? I know nothing ~ Jon Snow

### CODE REVIEW

File: test/integration/drag_and_drop_test.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
+
+class DragAndDropTest < ActionDispatch::IntegrationTest
+  before do
+    Capybara.current_driver = :poltergeist
+    @user = create :user
+    log_in_as @user
+
+    4.times do |i|
```
Comment (colinxfleming) [inline test/integration/drag_and_drop_test.rb]: Took a look at this and I was having it choke on the `before` block -- `table_content` uses some stuff on the pregnancy object (specifically LMP), and pregnancy objects didn't seem to be being created. 

This is probably a signal that we should either a) adjust the factory to create pregnancy when we create a patient or b) write a validation such that making a patient without a pregnancy chokes. Or maybe neither! Thoughts?

Comment (colinxfleming) [inline test/integration/drag_and_drop_test.rb]: Anyway, here's a before block that'll work:

```
  before do
    Capybara.current_driver = :poltergeist
    @user = create :user
    log_in_as @user

    4.times do |i|
      pt = create :patient, name: "Patient #{i}",
                       primary_phone: "123-123-123#{i}",
                       created_by: @user
      create :pregnancy, patient: pt
    end

    @user.add_patient Patient.find_by(name: 'Patient 0')
    @user.add_patient Patient.find_by(name: 'Patient 1')
    @user.add_patient Patient.find_by(name: 'Patient 2')
    @user.add_patient Patient.find_by(name: 'Patient 3')
  end
```

Visiting `dashboard_path` then shows you this: 
image

gl hf

Comment (Kevin-Wei) [inline test/integration/drag_and_drop_test.rb]: as usual setup failure. actual test super EZ thanks @colinxfleming

**Comment by colinxfleming** [comment]
Thanks a ton for finishing this off, @Kevin-Wei ! $$$

**Pull request merged**